localBuildRun.ps1

```diff
@@ -5,4 +5,4 @@ Copy-Item -Force -Recurse -Path $PSScriptRoot\environments\ -Destination $PSScri
 Copy-Item -Force -Recurse -Path $PSScriptRoot\runners\ -Destination $PSScriptRoot\build
 npm test
 if(-not $?) { throw 'tests failed' } 
-node $PSScriptRoot\build\engine\run.js $args
\ No newline at end of file
+node $PSScriptRoot\build\engine\run.js $args 
```

Comment:
If we wanted to have '--commentErrors' hardcoded into the localBuildRun.sh file. Shouldn't we add it to  localBuildRun.ps1, buildRun.sh, and BuildRun.ps1 too?